src/module/item/item.js

```diff
@@ -974,6 +1028,44 @@ export class ItemSFRPG extends Item {
         });
     }
 
+    async _rollVehicleDamage({ event } = {}) {
+        const itemData = this.data.data;
+
+        if (!this.hasDamage) {
+            throw new Error("you may not make a Damage Roll with this item");
```

Comment:
Please replace this with ui.notifications.error() call with a nice localized error message.